snippets/snippets.json

```diff
@@ -1,19 +1,92 @@
 {
     "source.jenkins": {
-        "template": {
-            "prefix" : "tmp",
-            "body": "pipeline {\n\tagent any\n\tstages {\n\t\tstage('${1:stage}') {\n\t\t\tsteps {\n\t\t\t\tstep {\n\t\t\t}\n\t\t}\n\t}\n}",
-            "description": "basic template"
+        {
+        "pipeline": {
+            "prefix": "pipeline",
+            "body": [
+              "pipeline{",
+              "    agent{",
+              "        label \"${node}\"",
+              "    }",
+              "    stages{",
+              "        stage(\"${A}\"){",
+              "            steps{",
+              "                echo \"========${something}========\"",
```

Comment:
I would leave only word `something` there, it's easier to edit it by going through them one by one with tabulator

also if you duplicate word inside like you did with `something` when I try to change one, all of them are changing